docs/ABOUT.md

```diff
@@ -1,3 +1,8 @@
-Scala is a object-oriented, functional language that runs on the Java Virtual Machine (JVM).  It was created by Martin Odersky.
+Scala combines object-oriented and functional programming in one concise, high-level language. Scala's static types help avoid bugs in complex applications, and its JVM and JavaScript runtimes let you build high-performance systems with easy access to huge ecosystems of libraries.
 
-Scala is a portmanteau of “Scalable Language”, which is said to mean (by the official website) that the language scales with you.
+* Features
+    * JVM Interoperability - Scala runs on the JVM, so Java and Scala stacks can be mixed for seamless integration.
+    * Type Inference - Scala is statically typed. But, Scala provides type inference to ease the burden of an advanced type system.
```

Comment:
`But, Scala provides` might be rephrased to `However, Scala provides`

Reply:
Agreed. This is much better. I will make this change.
Thanks